themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -117,7 +117,7 @@
     content="{{- $.Params.summary | default $.Site.Params.description | markdownify -}}"
   />
   <meta property="og:type" content="{{- $.Scratch.Get "pagetype" -}}" />
-  <meta property="og:url" content="{{- .Permalink -}}" />
+  <meta property="og:url" content="{{- .Permalink | absURL -}}" />
```

Comment:
This is showing up as 

```
<meta property="og:url" content="/preview/gsa/digitalgov.gov/nl-fix-og-url/">
```

I'm expecting to see:

```
<meta property="og:url" content="https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-fix-og-url/preview/gsa/digitalgov.gov/nl-fix-og-url/">
```

To be clear, previous approval was based on me seeing what I expected to see. 
Now it's changed.

Reply:
This may require additional cloud.gov config.